resources/dns_acme_cert.rb

```diff
@@ -0,0 +1,71 @@
+require 'openssl'
```

Comment:
This should not be included here and should be done in an `action_class` block instead I think.